web-ui/src/components/notes/Note.jsx

```diff
@@ -86,11 +82,19 @@ const Notes = (props) => {
           Notes for {memberName}
         </h1>
         <div className="container">
-          <textarea
-            onChange={handleNoteChange}
-            value={note.description}
-          ></textarea>
-          <canvas ref={canvasRef}></canvas>
+          {isLoading ? (
+            <div className="skeleton">
+              <Skeleton variant="text" height={30} />
+              <Skeleton variant="text" height={30} />
+              <Skeleton variant="text" height={30} />
+              <Skeleton variant="text" height={30} />
```

Comment:
you might try a relative value like `1 rem` or whatever seems at the right scale. They tend to make for a better experience on a variety of devices.